background.js

```diff
@@ -1,156 +1,184 @@
 const filter = {
-    url: [
-        { hostEquals: 'teams.microsoft.com', pathPrefix: '/dl/launcher/' },
-        { hostSuffix: '.zoom.us', pathPrefix: '/j/' },
-        { pathEquals: '/SAML20/SP/ACS' },
-    ],
+  url: [
+    {hostEquals: 'teams.microsoft.com', pathPrefix: '/dl/launcher/'},
+    {hostSuffix: '.zoom.us', pathPrefix: '/j/'},
+    {pathEquals: '/SAML20/SP/ACS'},
+  ],
 };
 
-var configs = {
-    teams: {
-        enabled: true,
-        delay: 2
-    },
-    zoom: {
-        enabled: true,
-        delay: 2,
-        user_domains: [],
-        managed_domains: [],
-        domains: new Set()
-    },
-    globalprotect: {
-        enabled: true,
-        delay: 2,
-        user_domains: [],
-        managed_domains: [],
-        domains: new Set()
-    }
+const configs = {
+  teams: {
+    enabled: true,
+    delay: 2,
+  },
+  zoom: {
+    enabled: true,
+    delay: 2,
+    user_domains: [],
+    managed_domains: [],
+    domains: new Set(),
+  },
+  globalprotect: {
+    enabled: true,
+    delay: 2,
+    user_domains: [],
+    managed_domains: [],
+    domains: new Set(),
+  },
 };
 
-// sleep time expects seconds
+/**
+ * Sets a sleep interval to execute a command
+ * @param {int} time - number of seconds to wait
+ * @return {Promise}
+ */
 function sleep(time) {
-    // Convert sleep time in seconds to milliseconds for use in setTimeout
-    return new Promise((resolve) => setTimeout(resolve, time * 1000));
+  // Convert sleep time in seconds to milliseconds for use in setTimeout
+  return new Promise((resolve) => setTimeout(resolve, time * 1000));
```

Comment:
I realize this is a formatting only change, but I just happened to see the bug.

Because we're using Manifest V3, we should be using `chrome.alarms` instead of `setTimeout` which can be [canceled when this service is terminated](https://developer.chrome.com/docs/extensions/migrating/to-service-workers/#convert-timers).